warehouse/templates/manage/project/history.html

```diff
@@ -167,9 +172,16 @@ <h2>{% trans %}Security history{% endtrans %}</h2>
     {% elif event.tag == EventTag.Project.APITokenAdded %}
     <strong>{% trans %}API token added{% endtrans %}</strong>
     <small>{% trans %}Permissions: Can upload to this project{% endtrans %}</small>
+    {% if event.additional.user %}
     <small>
       {% trans %}Controlled by:{% endtrans %} <a href="{{ request.route_path('accounts.profile', username=event.additional.user) }}">{{ event.additional.user }}</a>
     </small>
+    {% endif %}
+    {% if event.additional.expires %}
+    <small>
+      {% trans %}Expiration: {% endtrans %} {{ humanize(event.additional.expires | ctime) }}
+    </small>
+    {% endif %}
```

Comment:
Might be nice to add an additional field here before this, something like:

```
Expiration Status: Active
```
then:
```
Expiration Status: Expired
```

If that's a ton of pain to compare times in the template though, we can omit this.

Reply:
I think that's doable with another Jinja2 filter; will do.

Reply:
[0fb4679](https://github.com/pypi/warehouse/pull/11272/commits/0fb4679ca2394e08389e8a284910bdcb33b7f95e)